cli/src/klio_cli/commands/job/gke.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2019-2020 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import logging
+import os
+import re
+
+import glom
+import yaml
+from kubernetes import client, config
+
+from klio_cli.commands import base
+from klio_cli.utils import docker_utils
+
+
+class GKECommandMixin(object):
```

Comment:
This seems to not use anything from the pipeline classes that it's mixed in with with the exception of `job_dir`. Does this need to be a mixin that's inherited? Could it just be a class representing/wrapping the k8s client? That gets instantiated with `job_dir` and provided to when one of the other pipeline classes are instantiated?